modules/meeting/app/controllers/concerns/meetings/agenda_component_streams.rb

```diff
@@ -86,42 +86,89 @@ def update_sidebar_participants_form_component_via_turbo_stream(meeting: @meetin
           component: Meetings::Sidebar::ParticipantsFormComponent.new(
             meeting:
           ),
-          status: :bad_request
+          status: :bad_request # TODO: why bad_request?
```

Comment:
We want to ensure a non-ok HTTP response so that the modal doesn't auto-close after the form submission. Do you have a specific other status in mind?